DependencyInjection/LiipImagineExtension.php

```diff
@@ -57,7 +57,7 @@ public function getConfiguration(array $config, ContainerBuilder $container)
     }
 
     /**
-     * @see Symfony\Component\DependencyInjection\Extension.ExtensionInterface::load()
+     * @see \Symfony\Component\DependencyInjection\Extension.ExtensionInterface::load()
```

Comment:
Excellent point, and there's more on how php docblocks FQCNs are supported here: https://phpdoc.org/docs/latest/guides/types.html#class-name